README.md

```diff
@@ -35,82 +35,146 @@ Connect joy via nano USB receiver to NUC and make sure it is in **DirectInput Mo
+
+### Manipulator button mapping
+
+|      Button      |           Function           |
+| :--------------: | :--------------------------: |
+|   `LEFT STICK`   | manipulator steering - `X Y` |
+|  `RIGHT STICK`   |  manipulator steering - `Z`  |
+|  `UP CROSS KEY`  |        increase speed        |
+| `DOWN CROSS KEY` |        decrease speed        |
+|       `A`        |       enable movement        |
+|       `B`        |        emergency stop        |
+|       `X`        |         linear mode          |
+|       `Y`        |         angular mode         |
+|       `LB`       |       disable movement       |
+|       `LT`       |        close gripper         |
+|       `RT`       |         open gripper         |
+
+To move the manipulator use sticks. By default, cartesian *linear/angular* movement in `X` and `Y` is held by the left stick and *linear/angular* `Z` movement is controlled with the right stick.
+
+### Robot button mapping
+
+|    Button     |            Function             |
+| :-----------: | :-----------------------------: |
+| `LEFT STICK`  | velocity steering - angular `Z` |
+| `RIGHT STICK` | velocity steering - linear `X`  |
+|     `LB`      |         enable driving          |
+|     `RB`      |        slow driving mode        |
+|     `RT`      |        fast driving mode        |
 
-## Running demo on Panther robot
+If neither `RB` nor `RT` is pressed, the robot operates in *regular* driving mode.
+
+---
+
+## Running on Panther robot
 
 1. Access NUC via ssh
 
-```bash
-ssh husarion@10.15.20.3
-```
+    ```bash
+    ssh husarion@10.15.20.3
+    ```
 
-default password is `husarion`
+    The default password is `husarion`
 
 2. Clone this repo
 
-```bash
-cd ~/husarion_ws/src/
-git clone https://github.com/husarion/kortex_joystick_demo.git
-```
+    ```bash
+    cd ~/husarion_ws/src/
+    git clone https://github.com/husarion/kortex_joystick_demo.git
+    ```
 
-3. Setup gamepad according to [Setup joy](#setup-joy)
+3. Setup a gamepad according to [Setup joy](#setup-joy)
 
 4. Setup virtual desktop
 
-```bash
-cd kortex_joystick_demo/
-. ./kortex_joystick_demo/scripts/setup_virtual_desktop.sh
-```
+    ```bash
+    cd kortex_joystick_demo/
+    . ./kortex_joystick_demo/scripts/setup_virtual_desktop.sh
+    ```
 
 5. Launch demo
 
-```bash
+    ```bash
+    ```
 
-docker compose -f compose.panther-kinova.yaml up
-```
+<!-- More info needed here TBD -->
 
-After successful launch open Web browser and go to [10.15.20.3:8080](http://10.15.20.3:8080/vnc_auto.html). Enter password (default `husarion`) and you should see RViz and Panther with manipulator.
+After a successful launch open the Web browser and go to [10.15.20.3:8080](http://10.15.20.3:8080/vnc_auto.html). Enter a password (default `husarion`) and you should see RViz and Panther with the manipulator.
 
-## Controlling Kinova manipulator
+<!-- --- -->
```

Comment:
The section below this line is not visible in Git. Is this intentional?

Reply:
Yep, same case as above

Reply:
:+1: